ckanext/harvester_data_qld_geoscience/templates/snippets/facet_list.html

```diff
@@ -12,9 +12,9 @@
         <ul class="{{ nav_class or 'list-unstyled nav nav-simple nav-facet' }}">
           {% for item in items %}
             {% set href = h.remove_url_param(name, item.name, extras=extras, alternative_url=alternative_url) if item.active else h.add_url_param(new_params={name: item.name}, extras=extras, alternative_url=alternative_url) %}
-            {# only change from base is label_function(item) to label_function(name, item)#}
+            {# only change from base is label_function(item) to label_function(name, item) and ensuring truncation #}
             {% set label = label_function(name, item) if label_function else item.display_name %}
-            {% set label_truncated = label|truncate(22) if not label_function else label %}
+            {% set label_truncated = label|truncate(22) %}
```

Comment:
did we have to 'hack' the side bar to split the data sources??

has ckan 2.10 or 2.11 made this easier to not need to do the split?

also what exactly are we needing to add/update/remove when compared to ckan core or our theme?

Reply:
From https://github.com/qld-gov-au/ckanext-harvester-data-qld-geoscience/blob/70d1ac02a18c86af7dda3827d1e993cfa1cfac88/ckanext/harvester_data_qld_geoscience/helpers.py#L5 it appears that our custom label function exists to convert the dataset types (`dataset` and `geoscience`) into domain names (`data.qld.gov.au` and `geoscience.data.qld.gov.au`).